VSTS/Update-ProjectVersionNumber.ps1

```diff
@@ -19,7 +20,7 @@ $header = @{Authorization = ("Basic {0}" -f $base64AuthInfo)}
 $buildDefs = Invoke-RestMethod -Uri $uri -Method Get -ContentType "application/json" -Headers $header
  
 # Find the build definition for this project
-$buildDef = $buildDefs.value | Where-Object { $_.Project.id -eq $ProjectId }
+$buildDef = $buildDefs.value | Where-Object { $_.Project.id -eq $ProjectId -and ($_.name -eq $buildName }
```

Comment:
This looks like there is no matching bracket for the one just before `$_.name`. I would suggest removing that, and the PR will be approved.